### ISSUE TITLE
Direwolf20 1.10 doesn't work.

### ISSUE THREAD

**Issue by spiral6** [Vazkii/CMPDL#7]
And considering the URL... it's pretty easy to guess why.

https://mods.curse.com/modpacks/minecraft/225578-ftb-presents-direwolf20

**Comment by Nincodedo** [comment]
Especially since that's the 1.7.10 version :P

https://mods.curse.com/modpacks/minecraft/253632-ftb-presents-direwolf20-1-10

Same issue tho.

**Comment by spiral6** [comment]
Ah right, my bad. Still, yeah.

**Comment by sp823** [comment]
Same issue here. I would think it would be an issue with any of the FTB packs since they do not have the normal project pages and instead link to the FTB site.

**Comment by Nincodedo** [comment]
Correct, and this isn't an easy fix since the FTB site doesn't have a download latest link like how Curse project pages do. You could use the Curse Widget API to get the URL for the latest file.

**Comment by sp823** [comment]
I ended up using the FTB launcher to download as a workaround, which still works for FTB packs - for now anyway. On Linux here so Curse client is not an option.

**Comment by robertmaxton42** [comment]
Wanted to re-open this - Do we have a solution yet? Would like to avoid trying to use the Curse client on my server, since I'm ssh'ing into it and Xming is slow as hell. (I think it *does* have a *.jar for linux now but I'd still prefer to avoid it.)

**Comment by blk-addr** [comment]
I've downloaded FTB modpacks successfully with [pull request 17](https://github.com/Vazkii/CMPDL/pull/17).  You'll need to specify the File ID however since there's no 'latest' link on their site.
